api/src/main/java/org/eclipse/microprofile/openapi/annotations/Components.java

```diff
@@ -105,4 +106,14 @@
      * @return the reusable Callback objects.
      */
     Callback[] callbacks() default {};
+
+    /**
+     * The list of optional extensions.
+     *
+     * @return an optional array of extensions
```

Comment:
I'm not sure if referring to extensions as "optional extensions" in the javadoc is a good idea. It's not clear in what way they're "optional".

In particular, it's not optional for implementers to include them in the model.

It is valid for the user not to use any extensions, so they're optional in that way, but that's true of many annotation parameters so I don't think it warrants calling it out here.

Reply:
I'll update this, thanks. It's originally from the text of `CallbackOperation#extensions` which already exists before this PR, but I think "optional" is not the best choice to describe this. (I'll update the `CallbackOperation` JavaDoc as well).

Reply:
Done